micromamba/src/shell.cpp

```diff
@@ -48,13 +48,13 @@ init_shell_parser(CLI::App* subcom)
                                                       "activate",
                                                       "deactivate",
                                                       "hook",
-                                                      "reactivate",
-                                                      "deactivate"
```

Comment:
I would expect this to break https://github.com/mamba-org/mamba/blob/bab7ae50d75f067c493b8d0cd065ea29bd3f9e8c/libmamba/data/mamba.sh#L69

Reply:
Hmm, here I am deleting a duplicate `deactivate` entry in a `std::set`, because it looks like a typo. Based on the usual definition of "set" I wouldn't expect this to have any effect. (In a mathematical set, an object is either a member or not, so it shouldn't matter how many times it occurs.)

Reply:
oh sorry I didn't saw the duplicate :)